test/workflow/scripts/dataset_cli.sh

```
@@ -0,0 +1,270 @@
+# Inspect Dataset
+echo "[Inspect Start] Inspect Dataset"
+osmo dataset inspect ${BUCKET_NAME}/${DATASET_NAME}_{{workflow_id}}
+if [ $? -ne 0 ]; then
+    echo "[Inspect Failed] Failed to inspect dataset"
+    fail=1
+fi
+echo "[Inspect Done]"
+
+# Delete Dataset
+echo "[Delete Start] Delete Dataset"
+osmo dataset delete ${BUCKET_NAME}/${DATASET_NAME}_{{workflow_id}} --force --all
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)